tests/zfs-tests/tests/functional/cli_root/zfs_receive/zfs_receive_raw_-d.ksh

```diff
@@ -0,0 +1,62 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2019 Datto, Inc. All rights reserved.
+#
+
+. $STF_SUITE/include/libtest.shlib
+
+#
+# DESCRIPTION:
+# zfs receive -f should create the expected encryption hierarchy.
```

Comment:
nit: sorry, i somehow missed this in my first review: i think this is supposed to be _zfs receive -d_

Reply:
Thats fine. I missed it when writing it :) fixed.